mp/src/game/client/momentum/ui/HUD/hud_mapinfo.cpp

```diff
@@ -109,16 +109,20 @@ void C_HudMapInfo::OnThink()
                                         m_wMainStart :
                                         CConstructLocalizedString(m_wBonusStart, m_pRunData->m_iCurrentTrack.Get()));
         }
-        else if (m_pRunData->m_iCurrentZone == 0) // End zone
+        else if (m_pRunData->m_iCurrentZone == 0) // End/cancel zone
         {
-            m_pMainStatusLabel->SetText(m_pRunData->m_iCurrentTrack == 0 ?
-                                        m_wMainEnd :
-                                        CConstructLocalizedString(m_wBonusEnd, m_pRunData->m_iCurrentTrack.Get()));
-        }
-        else if (m_pRunData->m_bMapFinished) // MOM_TODO does this even need to be here anymore? "Main End Zone" covers this
```

Comment:
Note: After testing I can confirm that this line did not need to be here, which is why I removed it in my PR

Reply:
Also, have no idea why this was posted as a review, I had selected "add single comment"